aspnet/SameSite/system-web-samesite.md

```diff
@@ -72,7 +72,7 @@ To target .NET 4.7.2 or later:
     <httpRuntime targetFramework="4.7.2"/>
   </system.web>
 
-* Verify the project file contains the correct [TargetFrameworkVersion](/visualstudio/msbuild/msbuild-target-framework-and-target-platform?view=vs-2019):
+* Verify the project file contains the correct [TargetFrameworkVersion](/visualstudio/msbuild/msbuild-target-framework-and-target-platform?view=vs-2019&preserve-view=true):
```

Comment:
Rather than adding the preserve view, the entire view parameter can be deleted. Everything from the question mark on can be deleted. This goes for all of the files in this PR. So, this one would be "/visualstudio/msbuild/msbuild-target-framework-and-target-platform"